task4/src/store/index.js

```diff
@@ -44,18 +47,24 @@ export default new Vuex.Store({
       state.username = nameData
       state.wallet = walletData
     },
-    setUserInfo(state, user) {
+    setUserInfo(state, { user, index }) {
       state.userNm = user.userName
       state.userWt = user.wallet
       state.userUd = user.uid
+      state.index = index
+      console.log(state.index)
     },
-    setWallet(state, nowWallet) {
+    setWallet(state, { nowWallet, userInfo }) {
       state.wallet = nowWallet
-      console.log(state.wallet)
+      for (let i=0; i<state.userList.length; i++) {
```

Comment:
`=` `<` の両外側にはスペースを空けるようにしましょう。
```suggestion
      for (let i = 0; i < state.userList.length; i++) {
```